src/Npgsql.GeoJSON/CrsMap.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+
+namespace Npgsql.GeoJSON
+{
+    struct CrsMapEntry
+    {
+        internal readonly int MinSrid;
+        internal readonly int MaxSrid;
+        internal readonly string Authority;
+
+        internal CrsMapEntry(int minSrid, int maxSrid, string authority)
+        {
+            MinSrid = minSrid;
+            MaxSrid = maxSrid;
+            Authority = string.IsInterned(authority) ?? authority;
+        }
+    }
+
+    struct CrsMapBuilder
```

Comment:
Mutable structs are always dangerous (it's very easy to introduce errors by copying)... Since this is only used once, in the construction of the `GeoJsonHandler`, consider moving this logic into `GeoJsonHandlerFactory`?

Reply:
I don't think so it's needed because it's designed only for the plugin with a few types inside. For large unobservable projects it would make sense, but not in that case.

Is it okay to use C# 7.2 and mark it as `ref struct`?